### PR TITLE
add s390 and ppc imagestream defs for mariadb

### PR DESCRIPTION
**Pull request by gabemontero**
See https://projects.engineering.redhat.com/browse/RHELPLAN-28399

@openshift/openshift-team-developer-experience @bparees @siamaksade fyi

### CODE REVIEW

File: official/mariadb/imagestreams/mariadb-rhel7.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
                     "iconClass": "icon-mariadb",
                     "openshift.io/display-name": "MariaDB (Latest)",
                     "openshift.io/provider-display-name": "Red Hat, Inc.",
-                    "tags": "database,mariadb"
+                    "tags": "database,mariadb,ppc64le"
```
Comment (bparees) [inline official/mariadb/imagestreams/mariadb-rhel7.json]: this looks odd to me?

Comment (bparees) [reply official/mariadb/imagestreams/mariadb-rhel7.json]: (likely a bug in the import code?)

Comment (gabemontero) [reply official/mariadb/imagestreams/mariadb-rhel7.json]: it does .... I think I see the problem (and realize I was not entirely aware of what `make import` did previously)

Comment (gabemontero) [reply official/mariadb/imagestreams/mariadb-rhel7.json]: interestingly enough, the same think existed with the hold ruby ppc entry that existed back in 3.10 .. see https://github.com/openshift/library/pull/190/files

Comment (gabemontero) [reply official/mariadb/imagestreams/mariadb-rhel7.json]: nope it was a problem in the new offical.yaml change ... fwiw the old 3.10 PR was wrong as well

**Pull request merged**